hilti/runtime/include/configuration.h

```diff
@@ -64,11 +65,34 @@ struct Configuration {
 };
 
 namespace configuration {
+
+namespace detail {
+/** The runtime's configuration singleton. */
+extern std::unique_ptr<hilti::rt::Configuration> __configuration;
+
+/**
+ * Returns the current global configuration without checking if it's already
+ * initialized. This is only safe to use if the runtime is already fully
+ * initialized, and should be left to internal use only where performance
+ * matters.
+ */
+inline const Configuration& unsafeGet() {
```

Comment:
I think there are a few users of `configuration::get` which could be `configuration::unsafeGet` now since outside of code changing the configuration we should be able to assume that the configuration has been set up by `rt::init`.

Reply:
Not I'd sure I'd generally switch but I went through and updates places where it could make a performance difference.